### ISSUE TITLE
Symfony 5 support

### ISSUE THREAD

**Issue by aykonsvk** [getsentry/sentry-symfony#289]
Sentry is not supporting latest 5.* version of Symfony Framework.

```
Restricting packages listed in "symfony/symfony" to "5.0.*"
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - sentry/sentry-symfony 3.2.1 requires symfony/config ^2.8||^3.0||^4.0 -> no matching package found.
    - sentry/sentry-symfony 3.2.0 requires symfony/config ^2.8||^3.0||^4.0 -> no matching package found.
    - Installation request for sentry/sentry-symfony ^3.2 -> satisfiable by sentry/sentry-symfony[3.2.0, 3.2.1].
```

**Comment by jmsche** [comment]
I'm not sure why, but the latest tag does not have the `5.0` version added: https://github.com/getsentry/sentry-symfony/blob/3.2.1/composer.json

**Comment by Briones** [comment]
It's already supported, but not released yet (I don't know why)

https://github.com/getsentry/sentry-symfony/issues/272

I'm also waiting to the release to be able to update some systems, in the midtime you can point to dev-master in your composer.json like this:

```
"sentry/sentry-symfony": "dev-master" 
```

**Comment by klaussilveira** [comment]
Any news on this @dcramer?

**Comment by HazAT** [comment]
Most devs are on vacation right now, we will deal with this early January. Please do not ping individual ppl this will not speed up the process.
Thanks!

**Comment by ruudk** [comment]
Any news on this?

**Comment by kavalanche** [comment]
Nah, they have no idea what the problem is.

**Comment by Jean85** [comment]
Sorry for the delay, 3.3.0 is out! https://github.com/getsentry/sentry-symfony/releases/tag/3.3.0

**Comment by pkruithof** [comment]
I think something went wrong, because there are no changes between 3.2.1 and 3.3.0:

https://github.com/getsentry/sentry-symfony/compare/3.3.0...3.2.1

**Comment by Jean85** [comment]
Yes, we already got reports about this: https://twitter.com/AlessandroLai/status/1217034197883375622

Something went wrong while releasing. We're fixing it, it seems correct now: https://github.com/getsentry/sentry-symfony/compare/3.2.1...3.3.0

**Comment by pkruithof** [comment]
Thanks for fixing and clarifying 👍

**Comment by Jean85** [comment]
Issue with the tag is tracked in #297, already solved with 3.3.1.
